### ISSUE TITLE
Bluetooth pause stops audio playback but doesn't actually pause Google Play Music

### ISSUE THREAD

**Issue by rubin110** [pdf/kdeconnect-chrome-extension#24]
I'm seeing (hearing?) this weird thing where pressing the pause button on my Bluetooth headphones will stop audio playback, but not pause Google Play Music. The audio looks like it's still playing (the pause/play button remains to show the pause symbol), track scrubber continues to move. After about 10 seconds my Bluetooth headphones start playing audio back again, with wherever the song is in Google Play Music.

Pausing the music with my keyboard or with just clicking the Pause/Play button does the right thing.

**Comment by rubin110** [comment]
Let me rephrase that. Simply using the pause/play button in the system tray, Google Play Music exhibits the same behavior.

**Comment by pdf** [comment]
I think you're reporting these problems to the wrong person.

**Comment by rubin110** [comment]
Yes, you're correct. Thanks!
